wpilibNewCommands/src/test/java/edu/wpi/first/wpilibj2/command/CommandScheduleTest.java

```diff
@@ -109,6 +111,102 @@ void schedulerCancelTest() {
     }
   }
 
+  @Test
+  void cancelNextCommandTest() {
+    try (CommandScheduler scheduler = new CommandScheduler()) {
+      Command[] commands = new Command[2];
+      var commandRunCounter = new AtomicInteger(0);
+      // for parity with C++ where the ordering of sets is non-deterministic, we cancel the other
+      // command in the
+      // first command and check that the second command is not run
```

Comment:
```suggestion
      // for parity with C++ where the ordering of sets is non-deterministic, we cancel the other
      // command in the first command and check that the second command is not run
```
Makes the comment easier to read.

Reply:
added in ada142a